backend/server.js

```diff
@@ -58,6 +58,32 @@ app.get("/billability-working-hours/:year/:years_back", (req, res) => {
   );
 });
 
+app.get("/salary", (req, res) => {
+  db.all(
+    "SELECT date FROM payday_payday WHERE deleted = 0 ORDER BY date DESC",
+    (err, rows) => {
+      if (err) {
+        res.status(400).json({ error: err.message });
+        return;
+      }
+      res.json({ data: rows });
+    }
+  );
+});
+
+app.get("/salary_report", (req, res) => {
+  db.all(
+    "SELECT tuntikirjaus_employee.lastname AS 'sukunimi', tuntikirjaus_employee.firstname AS 'etunimi', workphase_workphase.report_name AS 'raportin nimi', SUM(worklog_worklog.worker_hours) AS 'työtuntien summa' FROM worklog_worklog INNER JOIN payday_payday ON worklog_worklog.payday_id = payday_payday.id INNER JOIN workphase_workphase ON worklog_worklog.payday_id = workphase_workphase.id INNER JOIN tuntikirjaus_employee ON worklog_worklog.employee_id = tuntikirjaus_employee.id GROUP BY tuntikirjaus_employee.id, workphase_workphase.report_name ORDER BY tuntikirjaus_employee.lastname, tuntikirjaus_employee.firstname",
```

Comment:
Usually translations happen in the frontend, so you could keep the lastname, firstname etc as is instead of using for example ` tuntikirjaus_employee.lastname AS 'sukunimi'` to rename them here. Then in the table columns you can change the name that's shown in the UI.